.github/workflows/shellspec.yml

```diff
@@ -46,16 +46,25 @@ jobs:
       - if: ${{ matrix.os == 'ubuntu-latest' }}
         name: Run shellspec
         run: |
-          nix-shell -p shellspec --run 'shellspec'
+          sudo apt-get install -y ksh zsh make curl
```

Comment:
I should probably split this apart into install then run as well. Busy with some other tasks but can do this before EOD or around then, no rush this gets in before just did a red/green/refactor run on everything in prep for some other work relating to saving a debug tarball for scripts.